package.json

```diff
@@ -76,6 +77,7 @@
     "hack": "latest",
     "husky": "latest",
     "lint-staged": "latest",
+    "ncu": "~0.2.1",
```

Comment:
why is this necessary?

Reply:
As I was running the different npm scripts to try to get my local environment going, I noticed there was a call to this lib in the `npm run update` script, but it wasn't included as a dependency.  You know better what these scripts are for, so if this is useless I'll take it out and you can squash the commits when merging the PR

Reply:
oh, that's true, although the package name is `npm-check-updates`, let me update it 🙂